Packs/Uptycs/Integrations/Uptycs/Uptycs.py

```diff
@@ -367,12 +451,166 @@ def uptycs_get_assets_command():
     return entry
 
 
+def uptycs_get_detections():
+    """
+    return list of detections
+    """
+    http_method = 'get'
+    api_call = "/detections"
+
+    query_parameters = []
+    limit = demisto.args().get('limit')
+    if limit != -1 and limit is not None:
```

Comment:
```suggestion

    args = demisto.args()
    query_parameters = []
    limit = args.get('limit')
    if limit != -1 and limit is not None:
```

Let's switch this for all other places in the function that call demisto.args()

Reply:
updated as suggested